examples/wmts-layer-from-capabilities.js

```diff
@@ -24,7 +21,6 @@ $.ajax('data/WMTSCapabilities.xml').then(function(response) {
       }),
       new ol.layer.Tile({
         opacity: 1,
-        extent: projectionExtent,
```

Comment:
Setting an extent on the layer always prevents it from being rendered in multiple worlds.